README.md

```diff
@@ -39,7 +39,7 @@ class ExampleServer extends Springpress {
 }
 ```
 
-Don't forget to call `Springpress#listen` to binds and listens for connections. This method returns an [http.Server](https://nodejs.org/docs/latest-v16.x/api/http.html#class-httpserver) object (the built-in HTTP module). *You can use this instance in your testing framework to work with HTTP*.
+Don't forget to call `Springpress#listen` to bind and listen for connections. This method returns an [http.Server](https://nodejs.org/docs/latest-v16.x/api/http.html#class-httpserver) object (the built-in HTTP module). *You can use this instance in your testing framework to work with HTTP*.
```

Comment:
I think we should also change `bind and listen for` to `bind and listen to`.
Otherwise, LGTM👍

Reply:
@nnarct What's your opinion? I can't comment anything. My English is bad. 😢

Reply:
@eltfshr I think _listen_ in this case means listening until the Springpress notices some connections so **for** would be the best to describe that. But I don't exactly know how the Spingpress#listen works so I could be wrong maybe.

Reply:
I actually agree with you here, but the word _bind_ is there. I think maybe we change to `bind to and listen for`

Reply:
Exactly! That's all perfect , I think we should just go for that. Thank you